walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -26,6 +29,7 @@ This walkthrough demonstrates running the mobile showcase application using the
 [type=walkthroughResource]
 .Mobile Developer Services Documentation
 ****
+// TODO update
```

Comment:
the LA docs will be published on the RHMI splash page, not updating existing doc set